features/migration/impl/src/main/kotlin/io/element/android/features/migration/impl/MigrationPresenter.kt

```diff
@@ -68,6 +74,6 @@ class MigrationPresenter @Inject constructor(
     companion object {
         // Increment this value when you need to run the migration again, and
         // add step in the LaunchedEffect above
-        const val MIGRATION_VERSION = 1
+        const val MIGRATION_VERSION = 2
```

Comment:
`MIGRATION_VERSION` should maybe remove, and use `migrations.maxOf { it.order }` instead, maybe, again, I think it will reduce the risk of error.

For instance, the new code never write `MIGRATION_VERSION` to the store as before:

`migrationStore.setApplicationMigrationVersion(MIGRATION_VERSION)`